proto/profile.proto

```diff
@@ -129,10 +131,13 @@ message Breakdown {
   // profile start, this list should hold the values [3, 1, 2].
   repeated int64 time_offset_nanos = 1;
   // A list that is either empty or holding exactly the same number of elements
-  // as Sample.time_offset_nanos, associating each point in time with a
+  // as Breakdown.time_offset_nanos, associating each point in time with a
   // recorded value. For cumulative sample types the sum of these values should
   // add up to the corresponding Sample.value.
   repeated int64 value = 2;
+  // A list of ThreadInfo ids into the common table
+  // Either empty or holds the same number of elements as Breakdown.time_offset_nanos
+  repeated int64 thread_infos = 3;
```

Comment:
Ok, so the idea is to associate a `pid` / `tid` with each individual event (as listed in `time_offset_nanos`)?

I'm assuming the main motivation for adding this here (rather than on Sample.label) is to reduce pprof size?

Reply:
I've taken this idea and simplified it into a generic LabelSet over here: https://github.com/felixge/pprof/pull/3/files